spec/unit/recipes/default_spec.rb

```diff
@@ -5,18 +5,21 @@
 # Copyright:: 2018, BaritoLog.
 
 require 'spec_helper'
+require_relative '../../../libraries/yggdrasil'
 
 describe 'zookeeper::default' do
   context 'When all attributes are default, on Ubuntu 16.04' do
     let(:chef_run) do
       # for a complete list of available platforms and versions see:
       # https://github.com/customink/fauxhai/blob/master/PLATFORMS.md
+      mock_result = {zookeeper_hosts: ['zookeeper-01-ubuntu-1804.vagrantup.com', 'zookeeper-02-ubuntu-1804.vagrantup.com']}
```

Comment:
is this test working? you haven't specified yggdrasil in kitchen to be `true`
please create additional scenario in kitchen to test when yggdrasil is true